Debug App/Sources/View Controllers/MerchantSessionAndSettingsViewController.swift

```diff
@@ -601,10 +601,10 @@ class MerchantSessionAndSettingsViewController: UIViewController {
     @IBAction func primerHeadlessButtonTapped(_ sender: Any) {
         customDefinedApiKey = (apiKeyTextField.text ?? "").isEmpty ? nil : apiKeyTextField.text
 
-        let shippingOptions = applePayCaptureShippingDetails ?
-        PrimerApplePayOptions.ShippingOptions(isCaptureShippingAddressEnabled: true,
-                                              additionalShippingContactFields: [.name, .emailAddress, .phoneNumber],
-                                              requireShippingMethod: true) : nil
+//        let shippingOptions = applePayCaptureShippingDetails ?
```

Comment:
Leaving these commented in Debug App as we will be re-enabling when out of Beta

Reply:
Why are we doing this BTW? I miss the context...